metricbeat/module/ceph/command.go

```diff
@@ -0,0 +1,45 @@
+package ceph
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+func Execute(c *CephConfig, command string) (string, error) {
+
+	cmdArgs := []string{}
+	cmdArgsDefault := []string{"--conf", c.ConfigPath, "--name", c.User, "--format", "json"}
+	cmdArgs = append(cmdArgs, cmdArgsDefault...)
+	cmdArgs = append(cmdArgs, strings.Split(command, " ")...)
+
+	var output string
+
+	if strings.Contains(c.BinaryPath, "docker") {
+		cmdCeph := []string{"/usr/bin/ceph"}
```

Comment:
If is a container, I needed to call the ceph command inside him so that's why was different. In container, command will be something like "docker exec -i container_name /usr/bin/ceph ...". 
With REST api, I can remove this too.